README.md

```diff
@@ -219,6 +220,97 @@ The library adds a `form-submitted` to the submitted form. You can use it to sty
+  `DefaultControlErrorComponent`, and provide the requisite HTML template.
+
+  A common example is when using Ionic forms where each form field is wrapped in an `ion-item` and errors
+  are best displayed as a sibling `ion-item` of the field. Example below shows how this can be done using 
+  a custom control error component.
+
+  For example:
+  ```ts
+  // Custom error component that will replace the standard DefaultControlErrorComponent.
+  @Component({
+    template: `
+    <ion-item lines="none" class="ion-text-wrap" [class.hide-control]="hideError">
+      <ion-label color="danger" class="ion-no-margin ion-text-wrap" stacked>
+        {{ errorText }}
+      </ion-label>
+    </ion-item>
+    `
+  })
+  export class IonicControlErrorComponent extends DefaultControlErrorComponent {
+  }
+
+  @NgModule({
+    declarations: [AppComponent, IonicControlErrorComponent],
+    imports: [
+      ReactiveFormsModule,
+      ErrorTailorModule.forRoot({
+        errors: {
+          useValue: {
+            required: 'This field is required'
+          }
+        },
+        controlErrorComponent: IonicControlErrorComponent
+      })
+    ],
+    bootstrap: [AppComponent]
+  })
+  export class AppModule {}
+  ```
+- `controlErrorComponentAnchorFn` - Optional. A hook function that allows the error component's 
+  HTML element to be repositioned in the DOM. By default error components are inserted at the
+  bottom of the field with error. If your UI layout dictates a different positioning 
+  scheme, you may use this hook.
+
+  Since this error element can be placed anywhere in the DOM, it also has to be
+  removed when the error component is destroyed. To provide for this, this
+  function should return a callback that will then be invoked when the error component
+  is destroyed. You may use this to remove the error HTML element that you inserted
+  into the DOM yourself.
+
+  Example below shows how the Ionic specific error component is repositioned in the DOM
+  to suit Ionic's form layout. `hostElem` is the HTML element for the form control and
+  `errorElem` is the HTML element for the error component. 
+  ```ts
+  anchorIonicErrorComponent(hostElem: Element, errorElem: Element) {
+    hostElement.parentElement.insertAdjacentElement('afterend', errorElement);
+    return () => {
+      let errorNode = hostElement.parentElement.querySelector('custom-control-error');
+      if (errorNode) {
+        errorNode.remove();
+      }
+    };
+  }
+
+  @NgModule({
+    declarations: [AppComponent, IonicControlErrorComponent],
+    imports: [
+      ReactiveFormsModule,
+      ErrorTailorModule.forRoot({
+        errors: {
+          useValue: {
+            required: 'This field is required'
+          }
+        },
+        controlErrorComponent: IonicControlErrorComponent,
+        controlErrorComponentAnchorFn: anchorIonicErrorComponent
+      })
+    ],
+    bootstrap: [AppComponent]
+  })
+  export class AppModule {}
+  ```
+
+### Per Control Config
```

Comment:
Remove this section. It's in the inputs section.

Reply:
Got it. I also removed the redundant Global Config subsection.